test/CMakeLists.txt

```diff
@@ -57,10 +57,10 @@ set(MIOPEN_TEST_FLOAT_ARG)
 
 if(MIOPEN_TEST_HALF)   
     if(MIOPEN_BACKEND_OPENCL)
-        set(SKIP_TESTS test_gru test_rnn_vanilla test_lstm)
+        set(SKIP_TESTS test_gru test_rnn_vanilla test_lstm test_conv_for_dynamic_implicit_gemm)
     endif()
     if(MIOPEN_TEST_GFX908)
-       set(SKIP_TESTS test_immed_conv3d test_conv3d test_fusion_aux test_activation test_lrn_test test_ctc test_conv2d_bias test_conv3d_bias test_cba_inference test_cbna_inference test_pooling2d test_na_train test_na_inference test_bn_aux)
+       set(SKIP_TESTS test_immed_conv3d test_conv3d test_fusion_aux test_activation test_lrn_test test_ctc test_conv2d_bias test_conv3d_bias test_cba_inference test_cbna_inference test_pooling2d test_na_train test_na_inference test_bn_aux test_conv_for_dynamic_implicit_gemm)
```

Comment:
Could you please explain the reason. If reasoning is non-trivial, please add a comment to the cmake source.

Reply:
Hi @atamazov, this ctest dedicated for kernel in src/kernels/composable_kernel/asm/igemm_v4r1_dynamic.s, and it is currently only support gfx900, 906. Indeed it do can run on gfx908, but currently from daniel/chao's task, should add another kernel in the future for gfx908 case. So currently just skip this

Reply:
Okay. So I am expecting that you'll update `IsApplicable()` to enable for gfx900/906 only.